modules/jsdom/src/polyfills/glfw.ts

```diff
@@ -0,0 +1,425 @@
+// Copyright (c) 2021, NVIDIA CORPORATION.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {
+  glfw,
+  GLFWClientAPI,
+  GLFWContextCreationAPI,
+  GLFWInputMode,
+  GLFWModifierKey,
+  GLFWOpenGLProfile,
+  GLFWWindowAttribute,
+} from '@nvidia/glfw';
+import * as jsdom from 'jsdom';
+import {DOMWindow} from 'jsdom';
+import {Subscription} from 'rxjs';
+
+import {
+  isAltKey,
+  isCapsLock,
+  isCtrlKey,
+  isMetaKey,
+  isShiftKey
+} from '../../../glfw/src/events/event';
+
+import GLFW, {GLFWStandardCursor} from '../../../glfw/src/glfw';
```

Comment:
Looks like we need to export the `GLFWStandardCursor` enum and `CURSOR_NORMAL`, `CURSOR_HIDDEN`, and `CURSOR_DISABLED` constants in the C++ addon, then we can do this:

```suggestion
import {GLFWCursorState, GLFWStandardCursor} from '@nvidia/glfw';
```

Reply:
It looks like GLFWStandardCursor is actually a const and not an enum?
glfw ln 295: `export const GLFWStandardCursor = {...`
not sure how this one should work

Reply:
in `glfw/src/index.ts` we need an `export { GLFWStandardCursor } from './glfw';`